src/content/whats-new/2022/12/whats-new-12-08-errors-inbox-users-impacted-traces-in-context-slack.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Prioritize and address critical errors with new errors inbox features' 
+summary: 'Speed up error resolution and act on your error groups quickly with users impacted, traces in context, and Slack notifications at the entity level.' 
+releaseDate: '2022-12-08' 
+learnMoreLink: 'https://newrelic.com/blog/nerdlog/error-triage-workflow' 
+getStartedLink: 'https://docs.newrelic.com/docs/errors-inbox/errors-inbox/' 
+---
+You need rich, contextual data to ensure that you’re prioritizing and resolving the most mission critical errors quickly. Errors inbox has released a set of features that provide much-needed context to accelerate your error resolution workflows. 
```

Comment:
@andreasy0802 I changed "they're" to "you're" in the first sentence. If you meant "they're" could you clarify who that would be?